Content.Server/Atmos/GasMixture.cs

```diff
@@ -48,10 +50,11 @@ public float HeatCapacity
             get
             {
                 var capacity = 0f;
+                var atmosSystem = EntitySystem.Get<AtmosphereSystem>();
```

Comment:
From memory when I profiled using this variable here it adds on a bit to the perf but fixing the parallelisation is more important.